students/nartowska_karina/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,16 @@
+def capitalize(lower_case_word):
+    table_with_letters = [i for i in lower_case_word]
+    first_letter_ascii = ord(table_with_letters[0]) - 32
+    table_with_letters[0] = chr(first_letter_ascii)
+    print(''.join(table_with_letters), end=" ")
+
+
+def main():
+    print("Result:")
```

Comment:
I think this print could be located before ```for``` loop in line 11. Now, you have situations that "Result" is printed before the user input some value. Please remember such details next time :)